### PR TITLE
Adding tests for all workloads in Specstorage

### PR DESCRIPTION
**Pull request by AmarnatReddy**
# Description
Adding tests for all workloads in Specstorage
Logs: http://magna002.ceph.redhat.com/cephci-jenkins/cephci-run-4ZHE35/
Above logs are executed on baremetal only

Please include Automation development guidelines. Source of Test case - New Feature/Regression Test/Close loop of customer BZs
<details>

<summary>click to expand checklist</summary>

- [ ] Create a test case in Polarion reviewed and approved.
- [ ] Create a design/automation approach doc. Optional for tests with similar tests already automated.
- [ ] Review the automation design
- [ ] Implement the test script and perform test runs
- [ ] Submit PR for code review and approve
- [ ] Update Polarion Test with Automation script details and update automation fields
- [ ] If automation is part of Close loop, update BZ flag qe-test_coverage “+” and link Polarion test
</details>

### CODE REVIEW

File: suites/reef/cephfs/tier-2_cephfs_io.yaml [hunk 1]
```diff
@@ -227,11 +235,87 @@ tests:
...
+          Dir_count: 1
+          Files_per_dir: 1
+          File_size: 1k
+          Instances: 1
+  - test:
+      abort-on-fail: false
+      desc: "Run AI_IMAGE Workload"
+      module: cephfs_spec_storage.py
+      name: FS_SPEC_REPL_MAX_MDS_2_AI_IMAGE
+      polarion-id: "CEPH-XXXXX"
```
Comment (sumabai) [inline suites/reef/cephfs/tier-2_cephfs_io.yaml:281]: when are we planning to update polarion-id?

Comment (neha-gangadhar) [reply suites/reef/cephfs/tier-2_cephfs_io.yaml:293]: @AmarnatReddy Please add one polarion testcase for performance test. Other tests can have empty IDs. Lets not create multiple tests just for different workloads provided by SPECStorage.

Comment (AmarnatReddy) [reply suites/reef/cephfs/tier-2_cephfs_io.yaml:293]: I have created a ploarion id :  CEPH-83588102 and added it to the last test case

**Review by neha-gangadhar — APPROVED** [review]
lgtm

**Pull request merged**